go.mod

```diff
@@ -190,6 +184,7 @@ require (
 // replace directive is aligned with kcp v0.11.0 (https://github.com/kcp-dev/kcp/blob/v0.11.0/go.mod)
 replace (
 	github.com/google/cel-go => github.com/google/cel-go v0.12.6
+	github.com/kubestellar/kubestellar/space-framework => ./space-framework
```

Comment:
If I recall correctly, such a `replace` statement makes this `go.mod` impossible to reference in a `go.work`.

Reply:
I think this replace is needed for the initial merge + local work when you want to test changes on the local module. 
Once we push it to the repository, I believe we can remove that line completely.

Reply:
Also, I am unable to reconstruct the problem that I was vaguely remembering.  Maybe go tooling has gotten less picky since then.